source/catalog.html

```diff
@@ -0,0 +1,173 @@
+  </head>
+  <body>
+    <header class="page-header">
+      <div class="page-header__wrap">
+        <a class="page-header__logo" aria-label="Приложение Pink">
+          <img src='https://dummyimage.com/74x23.gif' alt="Приложение Pink"/>
+        </a>
+        <button class="page-header__toggle" type="button" title="Открыть меню"></button>
+        <nav class="page-header__nav">
+          <ul class="page-header__list nav-list">
+            <li class="nav-list__item">
+              <a class="nav-list__link" href="index.html">Главная</a>
+            </li>
+            <li class="nav-list__item">
+              <a class="nav-list__link nav-list__link--active">Фотографии</a>
+            </li>
+            <li class="nav-list__item">
+              <a class="nav-list__link" href="form.html">Конкурс</a>
+            </li>
+            <li class="nav-list__item">
+              <a class="nav-list__link" href="https://htmlacademy.ru/">HTML Academy</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <main class="page-main">
+      <section class="catalog">
+        <h1 class="catalog__title title">Фотографии пользователей</h1>
+        <p class="catalog__subtitle">Взгляните на фотографии, которые выкладывают пользователи! Видите, как не хватает ваших?</p>
+        <ul class="catalog__list card-list">
+          <li class="card-list__item card-list__item--primary">
+            <span class="card-list__sticker">Панорама дня</span>
+            <img class="card-list__image card-list__image--primary" src='https://dummyimage.com/320x137.gif' alt='Вид на реку с моста' width="320" height="137"/>
+            <div class="card-list__message user-blockquote">
+              <p class="user-blockquote__name">Анна <time class="user-blockquote__time" datetime="2020-11-18T00:00">(5 минут назад)</time></p>
+              <p class="user-blockquote__message">Прыгать или нет? Напишите в комментарии свой совет и смотрите прямую трансляцию в перископе, задавайте свои вопросы!</p>
+              <p class="user-blockquote__like">
+                <button class="user-blockquote__button" title="Мне нравиться"></button>
+                Нравится: 215
+              </p>
+            </div>
+          </li>
+          <li class="card-list__item">
+            <img class="card-list__image" src='https://dummyimage.com/280x180.gif' alt='Белый автомобиль под объективом' width="280" height="180"/>
+            <div class="card-list__message user-blockquote">
+              <p class="user-blockquote__name">Левон <time class="user-blockquote__time" datetime="2020-11-18T00:00">(5 часов назад)</time></p>
+              <p class="user-blockquote__message">Вчера он на луну летал, сегодня в руки к нам попал.</p>
+              <p class="user-blockquote__like">
+                <button class="user-blockquote__button" title="Мне нравиться"></button>
+                Нравится: 356
+              </p>
+            </div>
+          </li>
```

Comment:
По классам тут у тебя к самой карточке относится лишь содержимое **card-list__message**, остальное принадлежит списку, предлагаю сократить и упростить:

```suggestion
          <li class="card-list__item post">
            <img class="post__image" src='https://dummyimage.com/280x180.gif' alt='Белый автомобиль под объективом' width="280" height="180"/>
            <div class="post__content">
              <p class="post__user-name">Левон <time class="post__time" datetime="2020-11-18T00:00">(5 часов назад)</time></p>
              <p class="post__comment">Вчера он на луну летал, сегодня в руки к нам попал.</p>
              <p class="post__likes">
                <button class="post__like-button" title="Мне нравиться"></button>
                Нравится: 356
              </p>
            </div>
          </li>
```

Reply:
Вообще при нейминге старайся использовать короткие слова и конструкции. Длинные слова в будущем скажутся на читаемости:
```javascript
class="very-long-block__very-long-element very-long-block__very-long-element--very-long-modificator"
```
 Это утрированный пример, но все же подобные случаи встречаются. Все четко и по БЭМу, а читать сложно. В добавок объем кода растет

Reply:
Спасибо, в т.ч. за пример разметки, изучу побольше примеров разметки разных блоков